README.md

```diff
@@ -23,68 +23,74 @@ api:
   insecure: true
 ```
 
-Start Traefik:
+Start Traefik v3:
 
-```bash
+```sh
 docker run -d -p 8080:8080 -p 80:80 \
--v $PWD/traefik.yml:/etc/traefik/traefik.yml \
--v /var/run/docker.sock:/var/run/docker.sock \
-traefik:v2.5
-```
+  -v $PWD/traefik.yml:/etc/traefik/traefik.yml \
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  traefik:v3.2
+```  
 
-Start a backend server, named `test`:
+Start a backend server using the `traefik/whoami` image:
 
-```bash
+```sh
 docker run -d --name test traefik/whoami
 ```
 
-And finally, you can access to your `whoami` server throught Traefik, on the domain name `test.docker.localhost`:
+Access the whoami Service through Traefik:
 
 ```console
-# $ curl --header 'Host:test.docker.localhost' 'http://localhost:80/'
 $ curl test.docker.localhost
-Hostname: 390a880bdfab
+Hostname: 0693100b16de
 IP: 127.0.0.1
-IP: 172.17.0.3
+IP: ::1
+IP: 192.168.215.4
+RemoteAddr: 192.168.215.3:57618
 GET / HTTP/1.1
 Host: test.docker.localhost
-User-Agent: curl/7.65.3
+User-Agent: curl/8.7.1
 Accept: */*
 Accept-Encoding: gzip
-X-Forwarded-For: 172.17.0.1
+X-Forwarded-For: 192.168.215.1
 X-Forwarded-Host: test.docker.localhost
 X-Forwarded-Port: 80
 X-Forwarded-Proto: http
-X-Forwarded-Server: 7e073cb54211
-X-Real-Ip: 172.17.0.1
+X-Forwarded-Server: 8a37fd4f35fb
+X-Real-Ip: 192.168.215.1
 ```
 
-The web UI [http://localhost:8080](http://localhost:8080) will give you an overview of the routers, services, and middlewares.
+Access the Traefik Dashboard:
 
-![Web UI](https://raw.githubusercontent.com/traefik/traefik/v2.0/docs/content/assets/img/webui-dashboard.png)
+Open your web browser and navigate to `http://localhost:8080` to access the Traefik dashboard.
+This will provide an overview of routers, services, and middlewares.
 
-## Traefik v1 - Example usage
+![Web UI](https://raw.githubusercontent.com/traefik/traefik/v3.2/docs/content/assets/img/webui-dashboard.png)
 
-Grab a [sample configuration file](https://raw.githubusercontent.com/traefik/traefik/v1.7/traefik.sample.toml) and rename it to `traefik.toml`. Enable `docker` provider and web UI:
+## Traefik v2 - Example usage
```

Comment:
The wording and formatting between v2 and v3 are not identical.
Does it make sense to align them? (the new wording in v3 sounds good, so I would do the same for the v2 section)

Reply:
I think it does make sense, for consistency, I agree with you.

Reply:
Updated the PR @kevinpollet